packages/dev/inspector/src/components/actionTabs/tabs/propertyGrids/materials/pbrMaterialPropertyGridComponent.tsx

```diff
@@ -233,6 +233,7 @@ export class PBRMaterialPropertyGridComponent extends React.Component<IPBRMateri
             { label: "Sheen Reflectance", value: 85 },
             { label: "Luminance Over Alpha", value: 86 },
             { label: "Alpha", value: 87 },
+            { label: "Albedo Alpha", value: 88 }

```

Comment:
Ouch, we don't have constants/enum somewhere that defines all the possible values? @sebavan Would it make sense to add this so that users don't have to use raw numbers?

Reply:
This was meant to be internal only for the inspector :-) but if it starts to be reused outside, it would make total sense.